doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -477,7 +451,8 @@ Launch the RTVM
 
   .. code-block:: none
 
-     $ sudo /usr/share/acrn/samples/nuc/launch_hard_rt_vm.sh
+     $ cp /home/acrn/work/acrn-hyperviso/misc/vm_configs/sample_launch_scripts/nuc/launch_hard_rt_vm.sh  /usr/share/acrn/
```

Comment:
I believe you'll need to be `sudo` here?

Reply:
Thanks! Done!